opm/simulators/flow/TracerModel.hpp

```diff
@@ -139,21 +139,24 @@ class TracerModel : public GenericTracerModel<GetPropType<TypeTag, Properties::G
         for (std::size_t tracerIdx = 0; tracerIdx < this->tracerPhaseIdx_.size(); ++tracerIdx) {
             if (this->tracerPhaseIdx_[tracerIdx] == FluidSystem::waterPhaseIdx) {
                 if (! FluidSystem::phaseIsActive(FluidSystem::waterPhaseIdx)){
-                    throw std::runtime_error("Water tracer specified for non-water fluid system:" + this->name(tracerIdx));
+                    throw std::runtime_error("Water tracer specified for non-water fluid system:" +
+                                             this->name(tracerIdx));
```

Comment:
If you're changing this anyway then maybe we could add a trailing whitespace after the colon here:
`system:"` ➡️ `system: "`.

Reply:
indeed.